optionals/weapons_apex/CfgWeapons.hpp

```diff
@@ -0,0 +1,41 @@
+class CfgWeapons {
+    class arifle_MXM_Black_F;
+    class CLASS(tac_MXM_Mod): arifle_MXM_Black_F {
```

Comment:
Why does this have a `tac` midfix? `CLASS(x)` -> `tacs_x` already.

Reply:
Fixed, was an oversight.